modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,29 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+To register a client application to an existing API, the client application
+must first request access. When the request is approved by the API owner, a
+contract is created between the client application and the API, and the client
+application is registered.
 
-If terms and conditions are set for an asset, you are prompted to accept the terms before access can be granted.
+APIs that are protected by a client ID enforcement policy require client
+applications to provide a client ID and optional client secret. The client ID and client
+secret credentials are automatically created when the client application is
+registered.
+
+An API owner can use API Manager to view all API contracts ever created for an
+API instance. An API consumer can use Exchange's *My Applications* page to view
+the API contracts created between an API instance and that consumer's client
+applications.
+
+API providers can view, manage, and revoke these contracts with the steps in
+xref:api-manager::manage-client-apps-latest-task.adoc[Managing API Instance Contracts on API Manager].
+
+If terms and conditions are set for an asset, you are prompted to accept the terms before access is granted.
```

Comment:
How are these related to a contract? Are we saying that if the contract includes terms and conditions...?

Reply:
Terms & Conditions may be defined at organization level. If defined, then the contract stores the terms and conditions accepted when the contract was created.